gradle/mod.gradle

```diff
@@ -1241,58 +1241,6 @@ else
     }
 }
 
-if (project.hasProperty("usesCrowdin") && project.usesCrowdin.toString().toLowerCase() == "true" &&
-        System.getenv().containsKey("crowdinKey") && !project.gradle.startParameter.isOffline()) {
-    task downloadCrowdin() {
-        ext {
-            output = file('build/crowdin_raw.zip')
-            update = file('build/crowdin.json')
-            id = project.crowdinId
-        }
-        onlyIf {
-            System.getenv().containsKey("crowdinKey") && !project.gradle.startParameter.isOffline()
-        }
-        doLast {
-            download {
-                src "https://api.crowdin.com/api/project/${id}/export?key=${System.getenv("crowdinKey")}&json"
-                dest update
-                overwrite true
-            }
-            if (!update.text.contains('success')) {
-                throw new RuntimeException("Crowdin export failed, see ${update} for more info")
-            }
-            download {
-                src "https://api.crowdin.com/api/project/${id}/download/all.zip?key=${System.getenv("crowdinKey")}"
-                dest output
-                overwrite true
-            }
-        }
-    }
-
-    task crowdin(type: Zip) {
-        duplicatesStrategy = "include"
-        dependsOn downloadCrowdin
-        onlyIf {
-            !downloadCrowdin.state.skipped
-        }
-        archiveBaseName.set(project.name)
-        archiveVersion.set(project.version)
-        archiveClassifier.set('crowdin')
-        destinationDirectory = file('build/distributions')
-        from(zipTree(downloadCrowdin.output)){
-            eachFile {
-                if (!it.path.startsWith('assets/')) {
-                    it.relativePath = new RelativePath(true, it.relativePath.segments.drop(4))
-                }
-            }
-            exclude { it.isDirectory() }
-            rename { it.toLowerCase() }
-        }
-    }
-
-    tasks.processResources.dependsOn tasks.crowdin
-}
-
```

Comment:
There is no documentation or explanation why this is removed!

Reply:
Basically, as per discussions in Discord. This code was pretty much obsolete due to the fact that the crowdin plugin already handles this. For that reason I wouldn't bother and support this removed API in an attempt to fix it when the alternative already works.

Reply:
So with the update of the plugin it now works without these tasks?

Reply:
Yeah, the plugin introduces a few Gradle tasks: `crowdinDownloadProjectTranslation`, `crowdinCopyProjectTranslations`, which are used for downloading the source files, and they automatically land in the right location.

Additionally the uploading of source files was already done by this plugin by using `crowdinUploadSourceFiles`.
So I just changed over the API key to use the proper field to enable this plugin, and removed these custom tasks.